Community/Tdarr_Plugin_vdka_Tiered_CPU_CRF_Based_Configurable.js

```diff
@@ -126,6 +126,24 @@ const details = () => ({
         \\nExample:\\n
         true`,
     },
+    {
+    name: 'enable10bit',
+    type: 'boolean',
+    defaultValue: false,
+    inputUI: {
+      type: 'dropdown',
+      options: [
+        'false',
+        'true',
+      ],
+    },
+    tooltip: `Specify if output file should be 10bit. Default is false.
+                    \\nExample:\\n
+                    true
+
+                    \\nExample:\\n
+                    false`,
```

Comment:
Should probably add this forces 10 bit. If disabled it takes whatever the bit depth of the original file is.